packages/app/init/init.js

```diff
@@ -32,12 +34,7 @@ const init = (config) => {
       'Failed to process because the API key field is an empty string. Please provide an API key.'
     )
   } else {
-    return {
-      config: {
-        api_key: config.api_key,
-      },
-      name: 'default',
-    }
+    return new initializeConfig(config)
```

Comment:
Please change the name of `initializeConfig` here too.

Reply:
@faiq-naufal done at commit [f3459a2](https://github.com/inlivedev/inlive-js-sdk/pull/14/commits/f3459a280920e48847fd4effb60a07fbe81fa770)